Automattic/jetpack
# Removes all After the Deadline references from Jetpack

Fixes https://github.com/Automattic/wp-calypso/pull/31865
This complements https://github.com/Automattic/wp-calypso/pull/31880

Closes #9128
Closes #5640
Closes #3999
Closes #3506

#### Changes proposed in this Pull Request:

This PR removes *all* references for After the Deadline from Jetpack. This includes module, settings and the classic editor block.

| | Before | After |
| - | - | - |
| Writing Settings | ![Screen Shot 2019-03-29 at 19 14 34](https://user-images.githubusercontent.com/233601/55268093-a5e60300-5265-11e9-8fc3-4764c5fdb350.png) | ![Screen Shot 2019-03-29 at 19 19 52](https://user-images.githubusercontent.com/233601/55268099-af6f6b00-5265-11e9-93ad-9a9de4bb76ae.png) |
| Classic Editor Block | ![Screen Shot 2019-03-29 at 19 13 47](https://user-images.githubusercontent.com/233601/55268105-b8603c80-5265-11e9-8517-9324a0998d72.png) | ![Screen Shot 2019-03-29 at 19 21 22](https://user-images.githubusercontent.com/233601/55268115-c1e9a480-5265-11e9-9eaa-753e1bfde689.png) |


#### Testing instructions:

On an site with an active Jetpack connection:

* Go to _Jetpack_ > _Settings_ > _Writing_ and look for the _Composing_ card.
* Verify that the remaining options work as expected.

* Edit or Create a Post or Page with the Block Editor.
* Switch to the _Code Editor_ view.
* Add text to the code editor without adding any tags to activate the Classic Editor Block.
* Switch back to _Visual Editor_.
* Click on the Classic Editor Block and verify that all options work as expected.

#### Proposed changelog entry for your changes:

* After the Deadline proofreading service module removed
